easybuild/easyconfigs/p/python-mujoco/python-mujoco-2.2.2-foss-2022a.eb

```
@@ -0,0 +1,54 @@
+easyblock = 'PythonBundle'
+
+name = 'python-mujoco'
+version = '2.2.2'
+
+homepage = 'https://www.mujoco.org'
+description = """
+This package is the canonical Python bindings for the MuJoCo physics engine.
+The mujoco package provides direct access to raw MuJoCo C API functions, structs,
+constants, and enumerations. Structs are provided as Python classes, with
+Pythonic initialization and deletion semantics.
+"""
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+
+builddependencies = [
+    ('CMake', '3.24.3'),
+    ('Eigen', '3.4.0'),
+    ('pybind11', '2.9.2'),
+]
+
+dependencies = [
+    ('Python', '3.10.4'),
+    ('SciPy-bundle', '2022.05'),
+    ('Abseil', '20230125.2'),
+    ('MuJoCo', '2.2.2'),
+    ('GLFW', '3.3.8'),
+    ('PyOpenGL', '3.1.6'),
+]
+
+use_pip = True
+
+exts_list = [
+    ('mujoco', version, {
+        'patches': ['python-mujoco-2.2.2_use_eb_deps.patch'],
+        'preinstallopts': 'MUJOCO_PATH="$EBROOTMUJOCO"',
+        'checksums': [
+            {'mujoco-2.2.2.tar.gz': 'b1ca70b21ef01d0e56952d03a387bdae9c6b141c042379226596410a3ac82085'},
+            {'python-mujoco-2.2.2_use_eb_deps.patch':
+             'dc05de6a86abbc12e23becdcec4c04715c8dcb9abf3dfcdd6baf0113e22273c9'},
+        ],
+    }),
+    ('absl-py', '1.4.0', {
+        'modulename': 'absl',
+        'checksums': ['d2c244d01048ba476e7c080bd2c6df5e141d211de80223460d5b3b8a2a58433d'],
+    }),
+    ('glfw', '2.6.1', {
+        'checksums': ['e15c21f06e74cc9740bf36c3c618b5f0d015ef09fb4e216d138967d8ee532567'],
+    }),
+]
+
+sanity_pip_check = True
+
+moduleclass = 'lib'
```

Since `MuJoCo` itself is in `phys`, I'd use the same for the Python bindings:
```suggestion
moduleclass = 'phys'
```